src/ansys/fluent/core/services/meshing_queries.py

```diff
@@ -624,7 +624,16 @@ def __init__(self, service: MeshingQueriesService):
     docstring = None
 
     def GetFaceZoneAtLocation(self, location) -> Any:
-        """GetFaceZoneAtLocation."""
+        """
+        Return face zone at or closest to a specified location.
+
+        Note:  This function is not applicable to polyhedra meshes.
```

Comment:
```suggestion
        Note:  This function is not applicable to polyhedral meshes.
```

Reply:
The ansys doc is a typo or some other mistake

Reply:
Done. Thank you.